Gulpfile.js

```diff
@@ -107,6 +112,7 @@ const projects = [
         cwd: "packages/table/",
         dependencies: ["core"],
         copy: false,
+        isotest: true,
```

Comment:
i'm suggesting combining the _config flags_, not the test tasks. 
definitely should be separate gulp tasks.